web_responsive/models/inherited_res_users.py

```diff
@@ -11,3 +11,17 @@ class ResUsers(models.Model):
         ('normal', 'Normal'),
         ('sided', 'Sided'),
     ], string="Chatter Position", default='normal')
+
+
+    def __init__(self, pool, cr):
+        """ Override of __init__ to add access rights.
+        Access rights are disabled by default, but allowed on some specific
+        fields defined in self.SELF_{READ/WRITE}ABLE_FIELDS.
+        """
+        super(ResUsers, self).__init__(pool, cr)
+        # duplicate list to avoid modifying the original reference
+        type(self).SELF_WRITEABLE_FIELDS = list(self.SELF_WRITEABLE_FIELDS)
+        type(self).SELF_WRITEABLE_FIELDS.extend(['chatter_position'])
+        # duplicate list to avoid modifying the original reference
+        type(self).SELF_READABLE_FIELDS = list(self.SELF_READABLE_FIELDS)
+        type(self).SELF_READABLE_FIELDS.extend(['chatter_position'])
```

Comment:
TIL :smiley: 

I think it would be good to add a test for this. It would be so simple as impersonating the demo user and trying to write own field, and read if it got properly written.